ConsentViewController/Assets/javascript/SPJSReceiver.js

```diff
@@ -42,7 +42,7 @@ var getActionFromMessage = function (eventData) {
     return {
         id: String(choiceData.choice_id),
         type: choiceData.type,
-        pm_url: choiceData.iframe_url,
+        pmUrl: choiceData.pmUrl || choiceData.iframe_url,
```

Comment:
is this a new url to add different from before?

Reply:
ok, I saw below and I got the meaning

Reply:
just renaming the field :)